src/libs/PDCurses/sdl2_queue/pdcdisp.cpp

```diff
@@ -563,18 +563,17 @@ void PDC_doupdate(void)
 
 void PDC_pump_and_peep(void)
 {
-    SDL_Event event;
-
-    if (SDL_PollEvent(&event))
+    if (!pdc_event_queue.empty())
```

Comment:
(minor.. double negative)

Alternative comment could stick with the empty check and avoid a nesting level:
``` c++
if (pdc_event_queue.empty())
     return;
```